ionic/package.json

```
@@ -1,6 +1,6 @@
 {
-    "version": "7.7.0",
-    "name": "cordova-plugin-fcm-with-dependecy-updated",
+    "version": "7.8.1-penbase-beta-1",
```

```suggestion
    "version": "7.8.1-penbase",
```

Même commentaire sur le suffixe `beta-1`. A toi de voir si tu en as besoin et uniformiser.